region/maridia/inner-yellow/Watering Hole.json

```
@@ -142,6 +142,57 @@
         "Requires a pixel-perfect freeze in order to run onto and off of the frozen Zebs."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Frozen Zeb",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_frozenEnemyRunway",
+        {"canShineCharge": {"usedTiles": 13, "openEnd": 0}},
+        {"refill": ["Energy"]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt",
+        {"or": [
+          "Gravity",
+          {"disableEquipment": "Ice"}
+        ]}
```

with the choot, you dont need to disable ice, and you can get out of the water without gravity, with ice you dont need to wall jump or anything to get all the way out
```suggestion
        "canRModeSparkInterrupt"
```